pkg/util/testutil/helpers_test.go

```diff
@@ -18,3 +18,34 @@ func TestRandBytes(t *testing.T) {
 	assert.Len(t, bytes, 32)
 	assert.NotEmpty(t, bytes)
 }
+
+func TestCleanupCloser(t *testing.T) {
+	t.Parallel()
+
+	newCloser := func(c chan struct{}) closerFn {
+		return func() error {
+			c <- struct{}{}
+			return nil
+		}
+	}
+
+	c1 := make(chan struct{}, 1)
+	c2 := make(chan struct{}, 1)
+
+	// Test first add it's own Cleanup function which will
+	// assert that all Close method is being invoked
+	t.Cleanup(func() {
+		assert.Len(t, c1, 1)
```

Comment:
The assertion should be consistent with other tests, so the testify library should not be used.